roles/nf-core/tasks/main.yml

```diff
@@ -81,7 +81,7 @@
 - name: Download SILVA classifier data for ampliseq
   get_url:
     url: https://www.arb-silva.de/fileadmin/silva_databases/qiime/Silva_132_release.zip
-    dest: "{{ ngi_resources }}ampliseq/Silva_132_release.zip"
+    dest: "{{ ngi_rcontainers }}/ampliseq/Silva_132_release.zip"
```

Comment:
Should this be `ngi_rcontainers` or `ngi_containers`?

Reply:
Nice catch! Fixed it.